### PR TITLE
docs: update tutorial badge issuance turnaround time

### PR DESCRIPTION
**Pull request by mattrosno**
#### Changelog

**Changed**

- Swapped Kelly's name in Angular wrapping up to Matt
- Changed 2-4 review time to a couple weeks because we review badges weekly

### CODE REVIEW

File: src/pages/tutorial/angular/wrapping-up.mdx [hunk 1]
```diff
@@ -43,7 +43,7 @@ This badge demonstrates knowledge about Carbon's Angular components. To earn the
 
 ### FAQ and help
 
-Badge Issuer Contact: kellychurchill@us.ibm.com
+Badge Issuer Contact: matt.rosno@ibm.com
```
Comment (connor-leech) [inline src/pages/tutorial/angular/wrapping-up.mdx]:
```suggestion
Badge issuer contact: matt.rosno@ibm.com
```

We could also just write this as a sentence with a `mailto` link for your name.

```
Please don't hesitate to [email us](mailto:matt.rosno@ibm.com) with any additional questions.
```

Also wouldn't be the worst idea to use the Carbon team address to future-proof this.

File: src/pages/tutorial/angular/wrapping-up.mdx [hunk 2]
```diff
@@ -43,7 +43,7 @@ This badge demonstrates knowledge about Carbon's Angular components. To earn the
...
 
 Acclaim Support: For questions related to your Acclaim badge earner account and profile, as well as issues related to claiming your badge after receiving a notification, go to [http://support.youracclaim.com](http://support.youracclaim.com) for account issues.
```
Comment (connor-leech) [inline src/pages/tutorial/angular/wrapping-up.mdx]:
```suggestion
For questions related to your Acclaim badge earner account and profile, as well as issues related to claiming your badge after receiving a notification, go to [http://support.youracclaim.com](http://support.youracclaim.com) for account issues.
```

File: src/pages/tutorial/angular/wrapping-up.mdx [hunk 3]
```diff
@@ -67,6 +67,6 @@ Submit your completed IBM Digital Badge application by January 31, 2020, and rec
 
 ## Application
 
-Complete the [application form](https://www.surveygizmo.com/s3/5215497/IBM-Carbon-Design-System-Developer-Essentials-Angular) to request the digital badge and t-shirt. Once you submit the application, please allow 2-4 days for issuing the badge and 4-6 weeks to receive the t-shirt. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
+Complete the [application form](https://www.surveygizmo.com/s3/5215497/IBM-Carbon-Design-System-Developer-Essentials-Angular) to request the digital badge and t-shirt. Once you submit the application, please allow a couple weeks for issuing the badge and 4-6 weeks to receive the t-shirt. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
 
 _\* Shirts are available for a limited time while supplies last to U.S. shipping addresses and outside the U.S. to participating FED@IBM branches._
```
Comment (connor-leech) [inline src/pages/tutorial/angular/wrapping-up.mdx]: Is this still true?

File: src/pages/tutorial/react/wrapping-up.mdx [hunk 4]
```diff
@@ -32,7 +32,7 @@ Once you complete all five steps of the tutorial, you can apply for an [IBM Digi
 </Column>
 </Row>
 
-Once you submit the application, please allow two to four days for issuing the badge. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
+Once you submit the application, please allow a couple weeks for issuing the badge. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
```
Comment (connor-leech) [inline src/pages/tutorial/react/wrapping-up.mdx]:
```suggestion
Once you submit the application, please allow a few weeks for issuing the badge. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
```

File: src/pages/tutorial/angular/wrapping-up.mdx [hunk 5]
```diff
@@ -67,6 +67,6 @@ Submit your completed IBM Digital Badge application by January 31, 2020, and rec
 
 ## Application
 
-Complete the [application form](https://www.surveygizmo.com/s3/5215497/IBM-Carbon-Design-System-Developer-Essentials-Angular) to request the digital badge and t-shirt. Once you submit the application, please allow 2-4 days for issuing the badge and 4-6 weeks to receive the t-shirt. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
+Complete the [application form](https://www.surveygizmo.com/s3/5215497/IBM-Carbon-Design-System-Developer-Essentials-Angular) to request the digital badge and t-shirt. Once you submit the application, please allow a couple weeks for issuing the badge and 4-6 weeks to receive the t-shirt. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
```
Comment (connor-leech) [inline src/pages/tutorial/angular/wrapping-up.mdx]:
```suggestion
Complete the [application form](https://www.surveygizmo.com/s3/5215497/IBM-Carbon-Design-System-Developer-Essentials-Angular) to request the digital badge and t-shirt. Once you submit the application, please allow a few weeks for issuing the badge and 4-6 weeks to receive the t-shirt. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
```

File: src/pages/tutorial/vue/wrapping-up.mdx [hunk 6]
```diff
@@ -32,7 +32,7 @@ Once you complete all five steps of the tutorial, you can apply for an [IBM Digi
 </Column>
 </Row>
 
-Once you submit the application, please allow two to four days for issuing the badge. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
+Once you submit the application, please allow a couple weeks for issuing the badge. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
```
Comment (connor-leech) [inline src/pages/tutorial/vue/wrapping-up.mdx]:
```suggestion
Once you submit the application, please allow a few weeks for issuing the badge. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
```

File: src/pages/tutorial/angular/wrapping-up.mdx (hunk 3 above)
Comment (mattrosno) [reply src/pages/tutorial/angular/wrapping-up.mdx]: Yes, for Angular.

File: src/pages/tutorial/angular/wrapping-up.mdx (hunk 1 above)
Comment (mattrosno) [reply src/pages/tutorial/angular/wrapping-up.mdx]: Let's keep my name there until I get added to Carbon's team address.

**Pull request merged**